haskell/test/Test/Ligo/RegistryDAO.hs

```diff
@@ -128,9 +128,9 @@ initialStorage :: Address -> RegistryStorage
 initialStorage admin = let
   fs = baseDAORegistryStorageLigo { sExtra = def }
   in fs { sAdmin = admin, sConfig = (sConfig fs)
-            { cPeriod = 11
-            , cProposalFlushLevel = 22
-            , cProposalExpiredLevel = 100
+            { cPeriod = 25
+            , cProposalFlushLevel = 50
+            , cProposalExpiredLevel = 300
```

Comment:
:thinking: how is this change caused by the update of the dependencies?

Reply:
Had to change due to CI on-chain test failures.